nss-day-cohort-30/bangazon-api-ambitious-albatrosses
# Employee and Department Controllers and Tests

Fixes:
-N/A

Changes proposed in this request & reasons behind organizational or architectural decisions:
-Wrote controllers for employee and department resources
-Wrote tests for employee and department resources

Steps to test:
-STEP 1) Run the SQL from [this file](https://github.com/nss-day-cohort-30/bangazon-api-ambitious-albatrosses/blob/master/AmbitiousAlbatrossesData.txt) in Azure Data Studio
-STEP 2) Open Visual Studio Test Explorer and locate the TestEmployees and TestDepartments folders within the TestBangazonAPI folder.
-STEP 3) Select and Run all tests within the folders.

System configuration (3rd party libraries to be installed, command line utilities to run, UI instructions, expected outcome):
-N/A

Commit message:
-tests pass for CRUD functionality for employee and department resources

Link to feature ticket:
-[#6](https://github.com/nss-day-cohort-30/bangazon-api-ambitious-albatrosses/issues/6)
-[#7](https://github.com/nss-day-cohort-30/bangazon-api-ambitious-albatrosses/issues/7)


@ambitions-albatrosses
